idaes/models_extra/power_generation/costing/costing_dictionaries.py

```diff
@@ -66,3 +66,42 @@
 
 with open(os.path.join(directory, "sCO2_costing_parameters.json"), "r") as file:
```

Comment:
This code appears to be written at the module level, which means it will be run immediately on import (e.g. during testing). This is not a good idea and can have unexpected side effects. You should create a simple method to wrap this code which returns the costing dictionary.